demo/index.html

```diff
@@ -23,7 +23,7 @@ <h2 tabindex="0">Some interactions you should try</h2>
         With focus on the trigger, press Enter or Space to toggle the menu.
       </li>
       <li>
-        With the menu opened and focus on the trigger, press ArrowDown to move focus to the first menu item.
+        With the menu opened and focus on the trigger, press ArrowDown to move focus to the second menu item.
```

Comment:
I don't think this sentence is correct now. Instead, to the sentence above we could add a note:

> With focus on the trigger, press Enter or Space to toggle the menu. Focus will move to the first menu item.

Then maybe we could cut this sentence altogether.